code/modules/bitrunning/server/_parent.dm

```diff
@@ -49,12 +49,14 @@
 	/// Cooldown between being able to toggle broadcasting
 	COOLDOWN_DECLARE(broadcast_toggle_cd)
 
+
```

Comment:
i'm making double newlines a habit in dm, sorry

Reply:
you are evil